rialto_airflow/mais.py

```diff
@@ -50,43 +50,76 @@ def get_token(client_id: str, client_secret: str, base_url: str) -> str:
         raise TokenFetchError("Error during token request, see logs") from e
 
 
-def get_response(
-    access_token: str, url: str, params: Optional[dict[str, Any]] = None
-) -> dict[str, Any]:
+def get_response(access_token: str, url: str) -> dict[str, Any]:
     """Retrieves a JSON response from the MAIS ORCID API."""
     oauth_session = OAuth2Session(
         token={"access_token": access_token, "token_type": "Bearer"}
     )
-    response = oauth_session.get(url, params=params)
+    response = oauth_session.get(url)
     response.raise_for_status()  # Raise HTTPError for bad responses
     return response.json()
 
 
+def page_size() -> int:
+    """Returns the page size for the API request."""
+    return 100  # This is the max value from the API.
+
```

Comment:
this lets us easily alter the page size in the tests to verify paging works in UAT (which has less than 100 users)